src/redux/selectors/reportingRateSummary/parseTableData.js

```diff
@@ -0,0 +1,4 @@
+export default function parseTableData(state) {
+    console.log(state)
+    return null
```

Comment:
please add an `// @TODO` comment to this one, so we can search the project for `@TODO` once we think we're done. In my opinion breaking code should never make it into master without having something easy to find, like the comment above.